docs/core/extensions/includes/logging-samples-browser.md

```diff
@@ -0,0 +1,9 @@
+---
+author: IEvangelist
+ms.author: dapine
+ms.date: 07/30/2021
+ms.topic: include
+---
+
+> [!TIP]
+> All of the logging example source code is available in the **Samples Browser** for download. For more information, see [Browse code samples: Logging in .NET](/samples/dotnet/samples/csharp-logging-fundamentals).
```

Comment:
Not really for download (currently):

Related issues: https://github.com/dotnet/samples/issues/4723 https://github.com/dotnet/docs/issues/23380

Reply:
We are aware of this - and as @adegeo said here, https://github.com/dotnet/samples/issues/4723#issuecomment-890438052 it's being addressed internally soon.